python/paddle/fluid/framework.py

```diff
@@ -2002,6 +2003,160 @@ def replace_ellipsis(item):
 
         return self
 
+    def get_tensor(self, scope=None):
+        """
+        Get the tensor mapped in the scope. 
+
+        Parameters:
+            scope(Scope, optional) : If `scope` is None, it will be set to global scope 
+                obtained through 'global_scope()'. Otherwise, use `scope`.
+                Default: None
+
+        Retruns:
+            LoDTensor: the tensor mapped in the scope.
+
+        Examples:
+            .. code-block:: python
+
+            import paddle
+            import paddle.fluid as fluid
+            import numpy as np
+
+            paddle.enable_static()
+
+            x = fluid.data(name="x", shape=[10, 10], dtype='float32')
+
+            y = fluid.layers.fc(x, 10,name='fc')
+            prog = fluid.default_main_program()
+            place = paddle.CPUPlace()
+            exe = fluid.Executor(place)     
+            prog = paddle.static.default_main_program()                               
+            exe.run(fluid.default_startup_program())
+            inputs=np.ones((10,10),dtype='float32')
+            exe.run(prog,feed={'x':inputs},fetch_list=[y,])
+            path='temp/tensor_'
+            for var in prog.list_vars():
+                if var.persistable:
+                    t=var.get_tensor()
+                    paddle.save(t,path+var.name+'.pdtensor')
+
+            for var in prog.list_vars():
+                if var.persistable:
+                    t_load=paddle.load(path+var.name+'.pdtensor')
+                    var.set_tensor(t_load)
+        """
+        # The 'framework' is a low-level module, and 'executor' or 'core' 
+        # can not be imported at the begainning of this file. 
+        # Therefore, the above two modules are dynamically imported.
+        from .executor import global_scope
+        if scope is not None and not isinstance(scope, core._Scope):
+            raise TypeError(
+                "`scope` should be None or `core._Scope` type, but received {}.".
```

Comment:
use 2.0 whole API path, users don't core is which path, `core._Scope` -> `paddle.static.Scope`

Reply:
Done,thx.